pkg/defkinds/php/builder_test.go

```diff
@@ -363,13 +363,85 @@ func initBuildProdStageForAlpineImageTC(t *testing.T, mockCtrl *gomock.Controlle
 	}
 }
 
+func initBuildProdStageWithCacheMountsTC(t *testing.T, mockCtrl *gomock.Controller) buildTC {
+	genericDef := loadGenericDef(t, "testdata/build/zbuild.yml")
+	genericDef.RawLocks = loadDefLocks(t, "testdata/build/zbuild.lock")
+
+	solver := mocks.NewMockStateSolver(mockCtrl)
+
+	raw := loadRawTestdata(t, "testdata/composer/composer-symfony4.4.lock")
+	solver.EXPECT().FromContext(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
+	solver.EXPECT().ReadFile(
+		gomock.Any(), "composer.lock", gomock.Any(),
+	).Return(raw, nil)
+
+	kindHandler := php.NewPHPHandler()
+	kindHandler.WithSolver(solver)
+
+	return buildTC{
+		handler: kindHandler,
+		client:  llbtest.NewMockClient(mockCtrl),
+		buildOpts: builddef.BuildOpts{
+			Def:               &genericDef,
+			Stage:             "prod",
+			SessionID:         "<SESSION-ID>",
+			LocalUniqueID:     "x1htr02606a9rk8b0daewh9es",
+			EnableCacheMounts: true,
+			CacheIDNamespace:  "cache-ns",
+			BuildContext: &builddef.Context{
+				Source: "context",
+				Type:   builddef.ContextTypeLocal,
+			},
+		},
+		expectedState: "testdata/build/state-prod-with-cache-mounts.json",
+		expectedImage: &image.Image{
```

Comment:
souldn't the cache mount opt be asserted here somehow (i.e. in a more explicit way than the `json` state file only) ?
or is it enough to rely on the statefile (i.e. the state file describes this behavior) ?

Reply:
It's not super easy to see from json dumps, but yeah they're already asserting that cache mounts are actually mounted :wink:

For instance, with the following command:

```
$ cat pkg/llbutils/testdata/install-apt-packages-with-cache-mounts-but-no-layer-caching.json | \
    zbuild llbgraph --raw | \
    dot /dev/stdin -Tpng -o/dev/stdout | \
    feh -
```
![image](https://user-images.githubusercontent.com/557933/78542079-ee2fed80-77f6-11ea-9ad5-c2cae7272d5b.png)